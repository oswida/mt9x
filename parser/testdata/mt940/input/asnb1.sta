:20:0000000000
:25:NL81ASNB9999999999
:28C:1/1
:60F:C200101EUR444,29
:61:2001010101D65,00NODCNL47INGB9999999999
hr gjlm paulissen
:86:NL47INGB9999999999 hr gjlm paulissen

Betaling sieraden



:62F:C200101EUR379,29