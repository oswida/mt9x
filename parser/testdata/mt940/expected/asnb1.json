{
 "tag20": "0000000000",
 "tag25": {
  "account": "NL81ASNB9999999999"
 },
 "tag28": {
  "stmt_number": "1",
  "seq_number": "1"
 },
 "tag60": {
  "dc_mark": "C",
  "date": "2020-01-01T00:00:00Z",
  "currency": "EUR",
  "amount": "444.29"
 },
 "statements": [
  {
   "tag61": {
    "value_date": "2020-01-01T00:00:00Z",
    "entry_date": "0000-01-01T00:00:00Z",
    "dc_mark": "D",
    "amount": "65",
    "trx_ident": "NODC",
    "owner_ref": "NL47INGB9999999999",
    "details": "hr gjlm paulissen"
   },
   "tag86": [
    "NL47INGB9999999999 hr gjlm paulissen",
    "Betaling sieraden"
   ]
  }
 ],
 "tag62": {
  "dc_mark": "C",
  "date": "2020-01-01T00:00:00Z",
  "currency": "EUR",
  "amount": "379.29"
 }
}